hpcgap/lib/mapping.gi

```diff
@@ -168,17 +172,20 @@ InstallGlobalFunction( Image, function ( arg )
       map := arg[1];
       elm := arg[2];
 
+      # image of a single element <elm> under the mapping <map>
       if     FamSourceEqFamElm( FamilyObj( map ), FamilyObj( elm ) ) then
-
         if not IsMapping( map ) then
-          Error( "<map> must be a mapping" );
-        elif elm in Source( map ) then
-          return ImageElm( map, elm );
+          ErrorNoReturn( "<map> must be single-valued and total" );
+        elif not elm in Source( map ) then
+          ErrorNoReturn( "<elm> must be an element of Source(<map>)" );
         fi;
+        return ImageElm( map, elm );
 
       # image of a set or list of elments <elm> under the mapping <map>
-      elif     CollFamSourceEqFamElms( FamilyObj( map ), FamilyObj(elm) )
-           and IsSubset( Source( map ), elm ) then
+      elif CollFamSourceEqFamElms( FamilyObj( map ), FamilyObj(elm) ) then
+        if not IsSubset( Source( map ), elm ) then
+          ErrorNoReturn( "<coll> must be a subset of Source(<map>)" );
```

Comment:
Why is this `<coll>`? I expect, in this context the error message to be telling me the name of a variable here. But the variable here is called `<elm>`. So I think either you should say something like `The collection <elm>...`, or in this case (that `elm` is actually a collection), you could define a new variable `coll := elm` so that you can actually refer to `coll` in a proper way.

Reply:
I'm rewriting this to `<elm>`. I'll also change the wording from `a subset of Source..` to `contained in Source..`. Talking about "subsets" feels like it implies that `elm` and `Source..` are sets.

Reply:
Done.